attr_report.go

```diff
@@ -95,9 +102,14 @@ func DecodeVolumeMeasurement(b []byte) (VolumeMeasurement, error) {
 			v := native.Uint64(b[n:])
 			VolMeasurement.DownlinkPktNum = v
 			VolMeasurement.Flag |= DLNOP
+		case UR_VOLUME_MEASUREMENT_END:
```

Comment:
```
default:
    break
```

Reply:
break seems meaningless here, so I move return to here